src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/pact/provider/ManageRecallsUIPactTest.kt

```diff
@@ -103,17 +109,31 @@ class ManagerRecallsUiAuthorizedPactTest : ManagerRecallsUiPactTestBase() {
     "a recall does not exist",
     "a list of local delivery units exists",
     "a list of prisons exists",
-    "a list of courts exists"
+    "a list of courts exists",
+    "a missing user ID"
   )
   fun `no state required`() {
   }
 
   @State(
     "a recall exists",
-    "a fully populated recall exists"
+    "a fully populated recall exists",
+    "a user can be unassigned"
   )
   fun `a fully populated recall exists without documents`() {
-    recallRepository.save(fullyPopulatedRecall(::RecallId.zeroes()).copy(documents = emptySet()))
+    val assigneeUuid = UUID.fromString("11111111-1111-1111-1111-111111111111")
```

Comment:
FYI only: we shouldn't have to used a fixed value for this: our PACTs verify presence not values.